README.md

```diff
@@ -42,18 +82,16 @@ python setup.py install
 
 Note: If you have MemoryError in the install try to use `pip` with `--no-cache-dir`.
 
-*(Optional)* some advanced features (e.g. working pretrained models) requires extra packages, you can install it with:
+*(Optional)* some advanced features (e.g. working pretrained models or specific transforms) requires extra packages, you can install it with:
+
 ```bash
 pip install -r requirements.opt.txt
 ```
 
-Note:
-
-- some features require Python 3.5 and after (eg: Distributed multigpu, entmax)
-- we currently only support PyTorch 1.4
-
 ## Features
 
+- **/!\ New in OpenNMT-py 2.0**: [On the fly data processing]([here](https://opennmt.net/OpenNMT-py/FAQ.html#what-are-the-readily-available-on-the-fly-data-transforms).)
```

Comment:
/!\ rendered weird, maybe use an emoji as replacement to make it stand out?

Reply:
Following lines may need to be changed:

- [Conv2Conv convolution model]
- SRU "RNNs faster than CNN" paper

Reply:
I don't think these are broken by the new codepaths, are they?

Reply:
I mean we may add link to it if we use [] or term "paper".

Reply:
Oh, right. Will do.